frontend/app/components/navigation-button.tsx

```diff
@@ -0,0 +1,48 @@
+import type { ComponentProps } from 'react';
+
+import type { FlipProp, IconProp } from '@fortawesome/fontawesome-svg-core';
+import { faChevronRight } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+
+import { AppLink } from './links';
```

Comment:
This should be an aliased import (`~/`). I'm really considering writing an eslint plugin that checks for this.

Reply:
And I acknowledge that vscode is pretty terrible for this.. I often have to force push my PRs with fixes to the imports.